processor/batchprocessor/batch_processor.go

```diff
@@ -71,11 +73,12 @@ var _ consumer.TraceConsumer = (*batchTraceProcessor)(nil)
 var _ consumer.MetricsConsumer = (*batchMetricProcessor)(nil)
 
 // newBatchTracesProcessor creates a new batch processor that batches traces by size or with timeout
-func newBatchTracesProcessor(params component.ProcessorCreateParams, trace consumer.TraceConsumer, cfg *Config) *batchTraceProcessor {
+func newBatchTracesProcessor(params component.ProcessorCreateParams, trace consumer.TraceConsumer, cfg *Config, telemetryLevel telemetry.Level) *batchTraceProcessor {
```

Comment:
`telemetryLevel` could be added to the `Config` or perhaps `ProcessorCreateParams`?

Reply:
Please file an issue

Reply:
Done https://github.com/open-telemetry/opentelemetry-collector/issues/1413